### PR TITLE
Patch for Archotech PowerArmor

### PR DESCRIPTION
**Pull request by Airomeda**
## Changes

Describe adjustments to existing features made in this merge, e.g.
- adjusted armor value
- Costlist and WorkToMake balance

## Reasoning

Why did you choose to implement things this way, e.g.
- because archotech yes
## Testing

Check tests you have performed:
- [x] Compiles without warnings
- [x] Game runs without errors
- [x] (For compatibility patches) ...with and without patched mod loaded
- [x] Playtested a colony (specify how long)

### CODE REVIEW

File: Patches/Archotech PowerArmor/Apparel_JDS.xml [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                    </value>
+                </li>
+
+                <li Class="PatchOperationReplace">
+                    <xpath>Defs/ThingDef[@Name="Trooper_HelmetBase"]/statBases/MaxHitPoints</xpath>
+                    <value>
+                        <MaxHitPoints>360</MaxHitPoints>
+                        <Bulk>5.5</Bulk>
+                        <WornBulk>1.25</WornBulk>
+                        <ArmorRating_Heat>0.35</ArmorRating_Heat>
```
Comment (N7Huntsman) [inline Patches/Archotech PowerArmor/Apparel_JDS.xml:60]: Getting a duplicate of `ArmorRating_Heat`.

File: Patches/Archotech PowerArmor/Apparel_JDS.xml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+                    </value>
+                </li>
+
+                <li Class="PatchOperationReplace">
+                    <xpath>Defs/ThingDef[@Name="Trooper_ArmorBase"]/statBases/MaxHitPoints</xpath>
+                    <value>
+                        <MaxHitPoints>650</MaxHitPoints>
+                        <Bulk>100</Bulk>
+                        <WornBulk>15</WornBulk>
+                        <ArmorRating_Heat>0.50</ArmorRating_Heat>
```
Comment (N7Huntsman) [inline Patches/Archotech PowerArmor/Apparel_JDS.xml:131]: Getting a duplicate of `ArmorRating_Heat`.

File: Patches/Archotech PowerArmor/Apparel_JDS.xml [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+                    </value>
+                </li>
+
+                <li Class="PatchOperationAdd">
+                    <xpath>Defs/ThingDef[@Name="Trooper_HelmetBase"]/equippedStatOffsets</xpath>
+                    <value>
+                        <AimingAccuracy>0.20</AimingAccuracy>
+                        <ToxicSensitivity>-0.50</ToxicSensitivity>
+                        <SmokeSensitivity>-1</SmokeSensitivity>
+                        <NightVisionEfficiency_Apparel>0.85</NightVisionEfficiency_Apparel>
```
Comment (N7Huntsman) [inline Patches/Archotech PowerArmor/Apparel_JDS.xml:77]: It's Archotech gear, so you could probably increase this to 1.

**Review by N7Huntsman — CHANGES_REQUESTED** [review]
Minor things to fix.

File: Patches/Archotech PowerArmor/Apparel_JDS.xml (hunk 1 above)
Comment (Airomeda) [reply Patches/Archotech PowerArmor/Apparel_JDS.xml:60]: good catch, I will be fixing it right away.

**Review by N7Huntsman — APPROVED** [review]

**Pull request merged**